lib/block-supports/layout.php

```diff
@@ -461,6 +461,114 @@ function gutenberg_render_layout_support_flag( $block_content, $block ) {
 }
 add_filter( 'render_block', 'gutenberg_render_layout_support_flag', 10, 2 );
 
+/**
+ * Generates the CSS for layout position support from the style object.
+ *
+ * @param string $selector CSS selector.
+ * @param array  $style    Style object.
+ * @return string CSS styles on success. Else, empty string.
+ */
+function gutenberg_get_layout_position_style( $selector, $style ) {
+	$position_styles = array();
+	$position_type   = _wp_array_get( $style, array( 'layout', 'position' ), '' );
+
+	if (
+		in_array( $position_type, array( 'fixed', 'sticky' ), true )
+	) {
+		$sides = array( 'top', 'right', 'bottom', 'left' );
+
+		foreach ( $sides as $side ) {
+			$side_value = _wp_array_get( $style, array( 'layout', $side ) );
+			if ( null !== $side_value ) {
+				/*
+				* For fixed or sticky top positions,
+				* ensure the value includes an offset for the logged in admin bar.
+				*/
+				if (
+					'top' === $side &&
+					'fixed' === $position_type ||
+					'sticky' === $position_type
+				) {
+					// TODO: wrap the following value in a `calc()` + `$side_value`,
+					// so that any included value is treated as an offset.
```

Comment:
Will this mean that we can set `0px` as the offset for the bottom "side"?

Just thinking of the case where I'd want my footer to stick to the true bottom:

<img width="400" alt="Screen Shot 2022-10-28 at 3 49 50 pm" src="https://user-images.githubusercontent.com/6458278/198505533-a0b6a388-13e6-4983-a024-aed6569ecdd6.png">

Reply:
Oh, great catch! That particular `var()` usage with the toolbar height should only be applied on `top` — I'll fix it up 👍

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/44723/commits/5fe14685f233584f3e3dce1af7e1336b0f3636c3